js/app.js

```diff
@@ -1,129 +1,120 @@
+  //respond
+  alert(userName + 'your favorite animal is' + favAnimal);
+}
+questionFive();
 
 
 
 //6th question guess the number I am thinking
+function questionSix(){
+  let guessNum = 7;
+  let chances = 4;
 
-// let guessNum = 7;
-// let chances = 4;
-
-// while(chances > 0){
-//   chances = chances - 1;
-//   //prompt user for a number
-//   let userResponse = prompt('How many years did I serve in the MN National Guard?');
-//   console.log('userResponse is a string ', typeof(userResponse));
+  while(chances > 0){
+    chances = chances - 1;
+    //prompt user for a number
+    let userResponse = prompt('How many years did I serve in the MN National Guard?');
+    console.log('userResponse is a string ', typeof(userResponse));
 
-//   let userNumber =  Number(userResponse);
-//   console.log('userResponse is a num ', userNumber, typeof(userNumber));
+    let userNumber =  Number(userResponse);
+    console.log('userResponse is a num ', userNumber, typeof(userNumber));
 
-//   if (!userResponse){
-//     break; //incase no input
-//   }
+    if (!userResponse){
+      break; //incase no input
+    }
 
-//   //user answer is correct handle
-//   if(userNumber === guessNum){
-//     alert(userName + ' Well done!');
-//     chances = 0;
-//   }
+    //user answer is correct handle
+    if(userNumber === guessNum){
+      alert(userName + ' Well done!');
+      chances = 0;
+    }
 
 
-//   //handle too high or too low
-//   if(userNumber < guessNum){
-//     alert(userName + 'you are too low, keep trying!');
-//   }
+    //handle too high or too low
+    if(userNumber < guessNum){
+      alert(userName + 'you are too low, keep trying!');
+    }
 
-//   //too high if
-// }//closes while
-// alert('the correct answer is ' + guessNum);
+  //too high if
+  }//closes while
+  alert('the correct answer is ' + guessNum);
+}
+questionSix();
 
 
 
 //create an array
 
-let states = ['ILLINOIS','IOWA','ARKANSAS','CALIFORNIA','TEXAS','OMAN','GERMANY'];
-console.log(states.length);
-
-let attempts = 0;
-let correctAnswer = 0;
-
-while(attempts !== 7){
-  let userAnswer = prompt('What states of the world have I lived in?');
-  let userAnswerTrimmedToUpperCase = userAnswer.toUpperCase().trim();
-  console.log('userAnswerTrimmedToUpperCase',userAnswerTrimmedToUpperCase);
-
-  if(states.indexOf(userAnswerTrimmedToUpperCase) >= 0){
-    alert('Yes! ' + userAnswerTrimmedToUpperCase + ' is one of my states of residence.');
-    correctAnswer = correctAnswer + 1;
-    // correctAnswer += 1;
-    // correctAnswer++;
-  } else {
-    alert('Your wrong@!!#%^#%%^');
-  }
-
-<<<<<<< HEAD
-  //too high if
-}//closes while
-alert('the correct answer is ' + guessNum);
-let states = ['poke bowl','salmon chazuke','rice','brats','noodles','sushi','coffee'];
-console.log(states.length);
+let favoriteFood = ['poke bowl','salmon chazuke','rice','brats','noodles','sushi','coffee'];
+console.log(favoriteFood.length);
 
 let attempts = 0;
 let correctAnswer = 0;
```

Comment:
One thing I notice is you're not adding points to correctAnswer until favoriteFoods, so a user's correct answers beforehand wont be counted towards their total